pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -464,6 +466,12 @@ stdenv.mkDerivation ({
     runHook postCheck
   '';
 
+  benchmarkPhase = ''
+    runHook preBenchmark
+    ${setupCommand} bench ${benchTarget} ${benchFlags}
```

Comment:
`testFlags` are passed to the test binary itself via `--test-option`, we should do this here as well for consistency.